script.js

```diff
@@ -8,10 +15,10 @@ fetch('https://flipsum-ipsum.net/api/icw/v1/generate?ipsum=recipe-ipsum-text-gen
         //the line below replaces any accented character by the matching non-accented character
         //e.g. "é" becomes "e"
         textOutput = textOutput.normalize("NFD").replace(/\p{Diacritic}/gu, "")
-        wordsArray =  textOutput.split(' ')
+        wordsArray = textOutput.split(' ')
```

Comment:
what is `wordsArray`? you havent declared it as a let or const and you appear to immediately discard it so could could just assign the split on the line below

Reply:
done